docs/content/guides/operations/providers/aws-provider/howto-aws-provider-irsa/index.md

```diff
@@ -0,0 +1,130 @@
+---
+type: docs
+title: "How-To: Configure the AWS cloud provider with IAM Roles for Service Accounts (IRSA)"
+linkTitle: "AWS provider with IAM Roles for Service Accounts(IRSA)"
+description: "Learn how to configure the AWS provider with IAM Roles for Service Accounts(IRSA) for your Radius Environment"
+weight: 300
+categories: "How-To"
+tags: ["AWS"]
+---
+
+The AWS provider allows you to deploy and connect to AWS resources from a Radius Environment on an EKS cluster. It can be configured:
+
+- [Interactively via `rad init`](#interactive-configuration)
+- [Manually via `rad env update` and `rad credential register`](#manual-configuration)
+
+## Prerequisites
+
+- [AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account) and an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html)
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+  - You will need the cluster's OIDC Issuer URL. [EKS Example](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html)
+- [Create an IAM Policy] (https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html)
+
+
+## Setup the AWS IAM Roles for Service Accounts(IRSA) for Radius
+
+To authorize Radius to connect to AWS using AWS IAM Roles for Service Accounts(IRSA), you should assign IAM roles to Kubernetes service accounts. 
+
+To associate an IAM role with a Kubernetes service account Create an IAM role and associate it with a Kubernetes service account.
+- Go to Identity and Access Management (IAM) on AWS portal and create a new role.
+![alt text](image.png)
```

Comment:
can you rename these images with a meaningful name

Reply:
Can you also use image shortcode to insert images. More details here - https://docs.radapp.io/contributing/docs/contributing-docs/#images

Reply:
updated the images to the specifies format